functions/subl.fish

```diff
@@ -31,7 +31,7 @@ function subl -d "Open Sublime Text"
 
   if set -e SUBL_PATH
     eval $SUBL_PATH $argv
-  else if begin; which subl > /dev/null 2>&1; and test -x (which subl); end
+  else if command -q subl
```

Comment:
Let's do `command -sq` instead. It's implicit now, but I believe (and [this commit seems to support](https://github.com/fish-shell/fish-shell/commit/dc0746bc451effbe56baf0d94d9cc8cf8d747762)) that `command -q` without the `--silent` used to error? If so, it was recent enough that we should explicitly add the `-s`.

Reply:
Good find, makes sense! Updated and fixed the commit message.